packages/Modal/src/Modal.tsx

```diff
@@ -209,12 +208,12 @@ const Modal: React.FunctionComponent<ModalProps> = (props: ModalProps) => {
                                     {title && <h5 className="ids-modal__title">{title}</h5>}
 
                                     <IconButton
-                                        size="small"
+                                        size="medium"
                                         className="ids-modal__close"
                                         onClick={onClose}
                                         appearance={{ type: "ghost", variant: "secondary" }}
                                         aria-label={closeBtnAriaLabel}
-                                        icon={<Close />}
+                                        icon={<DismissIcon />}
```

Comment:
In theorie these icons should only be used when the theme is workleap. Does it affect anything here?

Reply:
Same comment as above here too :P